src/background/services/storage.ts

```diff
@@ -1,16 +1,30 @@
-import type { Storage, StorageKey } from '@/shared/types'
+import type {
+  GrantDetails,
+  Storage,
+  StorageKey,
+  WalletAmount
+} from '@/shared/types'
 import { type Browser } from 'webextension-polyfill'
 import { EventsService } from './events'
 
 const defaultStorage = {
+  /**
+   * For migrations, increase this version and add a migration script in
+   * {@linkcode MIGRATIONS} below. New additions to structure that can be
```

Comment:
Aside: In VSCode, MIGRATIONS is linked (can be followed on click)
![image](https://github.com/interledger/web-monetization-extension/assets/8426945/36ce6d7d-0c3f-4ab2-8bad-c3790767fd0e)